package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "virtual_keyboard",
+  "version": "1.0.0",
+  "description": "$ npm install -g sass",
+  "main": "script.js",
+  "dependencies": {},
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.6.1",
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.14.3",
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "eslint-plugin-vue": "^5.2.3"
```

Comment:
You tried really hard to install `eslint`. Tried all the options) But you needed only 2 of these.